app/_gateway_entities/workspace.md

```diff
@@ -16,14 +16,136 @@ tier: enterprise
+  - q: Are there entities that can't exist in a Workspace?
+    a: |
+      Yes, some entities are global meaning they don't exist in any individual Workspace. For example `ca_certificates`.
+  - q: Can I use Workspaces in Konnect?
+    a: |
+      Konnect offers access to the more power Consumer Groups [/gateway/entities/consumer-group/]
+
+  - q: Can a Workspace share a name with another Workspace?
+    a: |
+      Individual Workspaces can be managed using [decK](/deck/). However, decK can't manage multiple Workspaces at the same time, or delete Workspaces.
+  
 ---
 
-TODO
 
-## What is a workspace?
+## What is a Workspace?
+
+Workspaces are a way of namespacing {{site.base_gateway}} entities so they can be managed independently. Workspaces maintain a unified routing table on the data plane to support client traffic segmentation.
+
+The data plane routes client traffic based on the configuration applied across all Workspaces. Configuring entities related to routing such as Services and Routes alter the client traffic routing behavior of the data plane but {{site.base_gateway}} will always attempt to ensure that routing rules don't contain conflicts. 
+
+{% mermaid %}
+flowchart LR
+    subgraph Workspace1 [Workspace-1]
+        A(Team A's Service)
+        B(Route)
+    end 
 
-This page is a stub
+    subgraph Workspace2 [Workspace-2]
+        C(Team B's - Service)
+        D(Route)
+
+    end
+
+    subgraph Workspace3 [Workspace-3]
+        E(Team C's - Service)
+        F(Route)
+    end
+    subgraph Gatewayorg [Gateway Organization]
+    Workspace1
+    Workspace2
+    Workspace3
+    end
+ 
+{% endmermaid %}
 
 ## Schema
 
 {% entity_schema %}
+
+
+### Conflict detection algorithm
+
+Routing rules are configured at the data plane level. To ensure that traffic can be routed to the appropriate Workspace, {{site.base_gateway}} uses a conflict detection algorithm. The algorithm can be explained like this: 
+
+When a Service or Route is **created** or **modified**, the {{site.base_gateway}} Router performs the following sequence to check for duplicate or matching rules: 
+
+1. If no matches found: The operation proceeds.
+2. If a match is found in the same Workspace: The operation proceeds
+3. If a match is found in a different Workspace: 
+  * If the matching Service or Route has no host value: issue a `409 Conflict` error
+  * If the host is a wildcard:
+        * if it matches issue a `409 Conflict` error
+        * If it doesn't match, the operation proceeds.
+  * If the host is an absolute value: Issue a `409 Conflict` error.
+
+## Roles, groups, and permissions
+
+Workspaces allow users to control {{site.base_gateway}} entities in isolation. Individual permissions for Workspaces are configured using RBAC. Roles are sets of permissions that can be assigned to admins and users and can be specific to a Workspace. There are two types of Admin roles in {{site.base_gateway}}: **Super Admin** and **Admin**. 
```

Comment:
```suggestion
Because Workspaces allow users to control {{site.base_gateway}} entities in isolation, users must have the correct permissions to configure a particular Workspace. Users will require either a Super Admin or Admin role to configure Workspaces. 

The following table details which Workspace permissions each Admin role has:
```